### PR TITLE
251 support for gazebo + mobile base

### LINKED ISSUE #251: Support for Gazebo + mobile base

### PR DESCRIPTION
**Pull request by RemiFabre**

### CODE REVIEW

**Comment by RemiFabre** [comment]
This was tested on a physical robot. The main features work but we've observed 2 problems:
- The LIDAR seems to have a weird behavior on Gazebo (self collision ?) -> TODO Rémi
- The SDK client now lists the wheels as joints which is not what we want. @simheo would you have a way to handle this?

**Comment by RemiFabre** [comment]
> This was tested on a physical robot. The main features work but we've observed 2 problems:
> 
> * The LIDAR seems to have a weird behavior on Gazebo (self collision ?) -> TODO Rémi
> * The SDK client now lists the wheels as joints which is not what we want. @simheo would you have a way to handle this?

-> Both problems are now fixed. This is now mergeable, waiting for review

File: mobile_base_controller/zuuu_description/launch/gazebo_simulation.launch.py [hunk 1]
```diff
@@ -136,10 +134,10 @@ def generate_launch_description():
     # Nodes to call
     nodes = [
         # joint_state_publisher_node,
-        joint_state_broadcaster_spawner,
+        # joint_state_broadcaster_spawner,
```
Comment (simheo) [inline mobile_base_controller/zuuu_description/launch/gazebo_simulation.launch.py:137]: We don't need these nodes anymore?

**Review by simheo — APPROVED** [review]

File: mobile_base_controller/zuuu_description/launch/gazebo_simulation.launch.py (hunk 1 above)
Comment (RemiFabre) [reply mobile_base_controller/zuuu_description/launch/gazebo_simulation.launch.py:137]: I tested with the most minimal setup possible but left stuff commented as it worked in the past and could help someone

**Pull request merged**